systemtest/src/test/java/io/strimzi/systemtest/rollingupdate/AlternativeReconcileTriggersST.java

```diff
@@ -433,7 +432,16 @@ void testAddingAndRemovingJbodVolumes(ExtensionContext extensionContext) {
 
     @BeforeAll
     void setup(ExtensionContext extensionContext) {
-        installClusterWideClusterOperator(extensionContext, NAMESPACE, CO_OPERATION_TIMEOUT_DEFAULT, RECONCILIATION_INTERVAL);
+        new Install.InstallBuilder()
```

Comment:
Why we somewhere assign the value to a variable and sometimes not? It's connected to test classes, where CO is redeployed during the tests?

Reply:
This is actually wrong in every instantiation of Install class we need to assign it to install instance because in `tearDownEnvForOperator` -> `install.deleteClusterOperatorInstallFiles();`

Reply:
So I should assign it to the instance.